pint/observatory/topo_obs.py

```diff
@@ -122,6 +127,16 @@ def gps_fullpath(self):
         return os.path.join(os.getenv('TEMPO2'),'clock',fname)
 
     @property
+    def bipm_fullpath(self):
+        """Returns full path to the TAI TT(BIPM) clock file.  Will first try PINT
+        data dirs, then fall back on $TEMPO2/clock."""
+        fname = 'tai2tt_bipm2015.clk'
```

Comment:
We get most of our clock_correction from TEMPO or Tempo2 by copying them into the datafile directory. We have not created those files yet. I don't know if we are going to in the future.